io/plugins/eu.esdihumboldt.hale.io.shp.test/src/eu/esdihumboldt/hale/io/shp/ShapefileInstanceWriterTest.groovy

```diff
@@ -293,16 +313,18 @@ class ShapefileInstanceWriterTest {
 					def jts = the_geom.geometry
 					assert jts instanceof Point
 					def name = inst.p.name.value()
-					assert name
-					switch (name) {
+
+					String dencodedString = new String(name.getBytes(shapeFileDataStore.getCharset()));
```

Comment:
Did you mean to write `decodedString`? In other methods as well  `decoded` is misspelt.

Reply:
Yes :-) I'll rename them